pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[build-system]
+requires = ["setuptools>=61.2", "wheel", "setuptools-git-versioning", ]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools-git-versioning]
+enabled = true
+template = "{tag}"
+dev_template = "{tag}.post{ccount}"
+dirty_template = "{tag}.post{ccount}+dirty"
+
+[project]
+name = "basin3d"
+dynamic = ["version"]
+authors = [{ name = "Valerie C Hendrix", email = "vchendrix@lbl.gov" },
+    { name = "Danielle Svehla Christianson", email = "dschristianson@lbl.gov" },
+    { name = "Charuleka Varadharajan", email = "cvaradharajan@lbl.gov" },
+    { name = "Catherine Wong", email = "catwong@lbl.gov" }]
+maintainers = [
+    { name = "Valerie C Hendrix", email = "vchendrix@lbl.gov" },
+    { name = "Danielle Svehla Christianson", email = "dschristianson@lbl.gov" },
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Programming Language :: Python",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Developers",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Hydrology"
+]
+description = "BASIN-3D Core Framework"
+requires-python = '>=3.8,<3.11'
+dependencies = [
+    "pyyaml>=6.0",
+    "requests",
+    "pydantic",
+    "tinydb",
+]
+
+
+[project.optional-dependencies]
+dev = ['pytest', 'pytest-flake8', 'pytest-mypy', 'pytest-cov',
```

Comment:
double checking that we still want to use pytest-flake8. I recall discussing that it had some issues and that straight flake8 was preferable?

Reply:
@dschristianson Thanks for pointing this out.  I think it is same to remove (and will do so).  We are not using this in our github actions.